src/test/java/dk/kb/present/transform/XSLTSolrTransformerTest.java

```diff
@@ -279,6 +281,19 @@ void testTitleExtraction() throws Exception {
 		//  assertTrue(solrString.contains("{\"id\":\""));
 	}
 
+	@Test
+	void testImageResource() throws Exception {
+
+		String solrString = TestUtil.getTransformedWithAccessFieldsAdded(MODS2SOLR, RECORD_40221e30);
+		// TODO: Add more detailed test
+
+		Gson gson = new GsonBuilder().setPrettyPrinting().create();
+		JsonElement je = JsonParser.parseString(solrString);
+		String prettyJsonString = gson.toJson(je);
+		System.out.println(prettyJsonString );
```

Comment:
Nothing is tested here!

Reply:
It test that XSLT does not fail and I guess the System.out was a good help when writing the XSLT.
All field test are new done in Solr.  Testing values in json is not very pretty since there is so much escaping involved.